src/frontend/src/pages/WorkPackageDetailPage/DeleteWorkPackageModalContainer/DeleteWorkPackageView.tsx

```diff
@@ -4,23 +4,12 @@
  */
 
 import { WbsNumber, wbsPipe } from 'shared';
-import {
-  Dialog,
-  DialogActions,
-  DialogContent,
-  DialogTitle,
-  FormControl,
-  FormLabel,
-  IconButton,
-  Typography
-} from '@mui/material';
+import { FormControl, FormLabel, Typography } from '@mui/material';
+import NERFormModal from '../../../components/NERFormModal';
 import { useForm } from 'react-hook-form';
-import NERSuccessButton from '../../../components/NERSuccessButton';
-import NERFailButton from '../../../components/NERFailButton';
 import ReactHookTextField from '../../../components/ReactHookTextField';
 import * as yup from 'yup';
 import { yupResolver } from '@hookform/resolvers/yup';
-import CloseIcon from '@mui/icons-material/Close';
 import { DeleteWorkPackageInputs } from './DeleteWorkPackage';
 
 interface DeleteWorkPackageViewProps {
```

Comment:
For the yup schema add a custom message for the required() section saying 'WBS number is required'